javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javassistmodel/JavassistAnnotationMemberDeclaration.java

```diff
@@ -0,0 +1,40 @@
+package com.github.javaparser.symbolsolver.javassistmodel;
+
+import com.github.javaparser.ast.expr.Expression;
+import com.github.javaparser.resolution.declarations.ResolvedAnnotationMemberDeclaration;
+import com.github.javaparser.resolution.types.ResolvedType;
+import com.github.javaparser.symbolsolver.model.resolution.TypeSolver;
+import javassist.CtMethod;
+
+/**
+ * @author Malte Skoruppa
+ */
+public class JavassistAnnotationMemberDeclaration implements ResolvedAnnotationMemberDeclaration {
+
+    private CtMethod annotationMember;
+    private TypeSolver typeSolver;
+
+    public JavassistAnnotationMemberDeclaration(CtMethod annotationMember, TypeSolver typeSolver) {
+        this.annotationMember = annotationMember;
+        this.typeSolver = typeSolver;
+    }
+
+    @Override
+    public Expression getDefaultValue() {
+        // TODO we should do something like this:
```

Comment:
Good, thanks for the comment

Reply:
Sure! I already made an issue about this one. :)